presto-native-execution/presto_cpp/main/tests/PrestoExchangeSourceTest.cpp

```diff
@@ -37,6 +39,25 @@ int main(int argc, char** argv) {
   return RUN_ALL_TESTS();
 }
 
+namespace {
+std::string getCertsPath(const std::string& fileName) {
+  std::string currentPath = fs::current_path().c_str();
+  if (boost::algorithm::ends_with(currentPath, "fbcode")) {
+    return currentPath +
+        "/github/presto-trunk/presto-native-execution/presto_cpp/main/tests/certs/" +
+        fileName;
+  }
+
+  // CLion runs the tests from cmake-build-release/ or cmake-build-debug/
+  // directory. Hard-coded json files are not copied there and test fails with
+  // file not found. Fixing the path so that we can trigger these tests from
+  // CLion.
+  boost::algorithm::replace_all(currentPath, "cmake-build-release/", "");
```

Comment:
On the Velox side, we have a similar problem with checked-in data files.
The solution there is to use `getExampleFilePath` which is similar to `getDataFilePath`
https://github.com/facebookincubator/velox/blob/f19d47c9186be0b765e1f418a58090b621cdaa58/velox/dwio/dwrf/test/OrcTest.h#L35
The expectation is to change the working directory in the CLion environment.